src/Ocean/SuperModels.jl

```diff
@@ -0,0 +1,209 @@
+module SuperModels
+
+using ClimateMachine
+
+using ...DGMethods.NumericalFluxes
+
+using ..HydrostaticBoussinesq:
+    HydrostaticBoussinesqModel, NonLinearAdvectionTerm
+
+using ..OceanProblems: InitialValueProblem
+using ..Domains: array_type, communicator
+using ..Ocean: FreeSlip, Impenetrable, Insulating, OceanBC, Penetrable
+using ..Ocean.Fields: SpectralElementField
+
+using ...Mesh.Filters: CutoffFilter, ExponentialFilter
+using ...Mesh.Grids: polynomialorder
+
+using ClimateMachine:
+    LSRK144NiegemannDiehlBusch,
+    OceanBoxGCMConfigType,
+    OceanBoxGCMSpecificInfo,
+    DriverConfiguration
+
+import ClimateMachine: SolverConfiguration
+
+#####
+##### Build default EarthParameters
+#####
+
+using CLIMAParameters: AbstractEarthParameterSet
+
+struct EarthParameters <: AbstractEarthParameterSet end
+
+#####
+##### It's super good
+#####
+
+struct HydrostaticBoussinesqSuperModel{D, E, S, F, N, T, C}
```

Comment:
Docs?

Reply:
I put docs on the constructor, so I'm going to leave docs off this so that `help? HydrostaticBoussinesqSuperModel` just returns the constructor docstring.